Assets/Resources/OVRBuildConfig.asset

```diff
@@ -12,6 +12,6 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 20553fac56ec59645857c0732b787431, type: 3}
   m_Name: OVRBuildConfig
   m_EditorClassIdentifier: 
-  androidSDKPath: C:\Users\DALI Lab\AppData\Local\Android\sdk
+  androidSDKPath: C:\Users\Cathy Wu\AppData\Local\Android\Sdk
```

Comment:
sad that this ends up being hardcoded - nothing to worry about but i guess you each have to reset this?